test/prism_regression/match_last_line.rb

```diff
@@ -1,11 +1,16 @@
 # typed: false
 
+if // # testing an empty pattern
+#  ^^ error: Unsupported node type `MatchCurLine`
+  end
```

Comment:
Maybe remove the redundant spaces?

```suggestion
end
```

Reply:
That's an accident, thanks